test/test_torch.py

```diff
@@ -6820,32 +6820,32 @@ def _test_helper(x, y, bias, memory_format):
                 _unsqueeze_op_add,
                 _unsqueeze_op_clone,
             ]
+            x_c = x.contiguous()
+            y_c = y.contiguous()
+            b_c = bias.contiguous()
             for fn in fns:
-                x_c = x.contiguous()
-                y_c = y.contiguous()
-                result_c = fn(x_c, y_c)
-                result = fn(x, y)
-                self.assertEqual(result, result_c)
+                is_inplace = '_(' in inspect.getsource(fn)
```

Comment:
This is pretty ugly but I get why it's being used. With our new OpInfo mechanism we have much nicer ways of running tests like this. Let me think about if we can use them instead here

Reply:
I'm not sure. The list of functions is a list of lambdas. It would be hard/ugly to rewrite that into something where you can query the underlying op. And this is Python which allows such things, so why not? ;)